server/aws-lsp-codewhisperer/src/language-server/netTransform/transformHandler.ts

```diff
@@ -0,0 +1,255 @@
+import path = require('path')
+import { Workspace } from '@aws/language-server-runtimes/out/features'
+import * as fs from 'fs'
+import * as os from 'os'
+import { v4 as uuidv4 } from 'uuid'
+import {
+    CreateUploadUrlResponse,
+    GetTransformationRequest,
+    StopTransformationRequest,
+} from '../../client/token/codewhispererbearertokenclient'
+import { CodeWhispererServiceToken } from '../codeWhispererService'
+import { getCWStartTransformRequest, getCWStartTransformResponse } from './converter'
+import {
+    CancellationJobStatus,
+    QNetCancelTransformRequest,
+    QNetCancelTransformResponse,
+    QNetGetTransformPlanRequest,
+    QNetGetTransformPlanResponse,
+    QNetGetTransformRequest,
+    QNetGetTransformResponse,
+    QNetStartTransformRequest,
+    QNetStartTransformResponse,
+} from './models'
+import { cleanup, createZip, getSha256 } from './utils'
+import fetch = require('node-fetch')
+
+export class TransformHandler {
+    private client: CodeWhispererServiceToken
+    private workspace: Workspace
+    constructor(client: CodeWhispererServiceToken, workspace: Workspace) {
+        this.client = client
+        this.workspace = workspace
+    }
+
+    async startTransformation(userInputrequest: QNetStartTransformRequest): Promise<QNetStartTransformResponse> {
+        try {
+            const uploadId = await this.preTransformationUploadCode(userInputrequest)
+            const request = getCWStartTransformRequest(userInputrequest, uploadId)
+            console.log('send request to start transform api: ' + JSON.stringify(request))
```

Comment:
This will break the server since it's listening on `STDOUT` of the process. And no-one will be able to read it since the output is not directed to any terminal or file.

You can use `features.logging` instead (https://github.com/aws/language-server-runtimes/blob/main/src/features/logging.ts)